src/tests/validity/function_optional_argument_array.ab

```diff
@@ -0,0 +1,13 @@
+// Output
+// 1100
+
+fun sum_array(a : [Num] = [Num]): Num {
+    let sum = 0
+    loop n in a {
+        sum += n;
+    }
+    return sum;
+}
+let x = [100,1000]
+let result = sum_array(x)
+echo "{1100}"
```

Comment:
Did you mean to echo a hard coded value?  I suggest:
```
let result = sum_array(x)
echo "{result}"
```

Reply:
You changed `echo "{1100}"` to `echo 1100`; this still doesn't test the behaviour of `sum_array(x)`.  Can you change it to `echo result` like the other tests?